src/rgw/rgw_main.cc

```diff
@@ -1,21 +1,9 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab
 
-#include <stdlib.h>
-#include <stdio.h>
-#include <string.h>
-#include <stdarg.h>
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <fcntl.h>
-#include <errno.h>
-#include <signal.h>
-
+extern "C" {
```

Comment:
@mikulely is current build broken without this change?

Reply:
No. As I've touched around, just take this chance to make `nm` happy.

Reply:
@mikulely could you be more specific regarding to the `nm` happy thing?

Reply:
https://stackoverflow.com/posts/67985/revisions

Reply:
@mikulely if you care about the exported symbols from `radosgw`, probably you should fix the CMakeLists.txt to avoid exporting them.

Reply:
@tchaikov Thx for minding. Just a trivial cleanup here, let's forget about it.

Reply:
i'd recommend do it right if we want to fix it even it's a cleanup.